examples/angular/package.json

```diff
@@ -13,7 +13,7 @@
     "@angular/platform-browser": "^12.0.0",
     "@angular/platform-browser-dynamic": "^12.0.0",
     "core-js": "^3.2.1",
-    "rxjs": "^7.3.0",
+    "rxjs": "^6.6.7",
```

Comment:
so, land #11924 first

Reply:
rxjs still needs a workaround as they've used `node` as condition for their CJS (and not `require`), but since we use the browser env we don't have that condition.

workaround: [`0e2c7d1` (#11961)](https://github.com/facebook/jest/pull/11961/commits/0e2c7d16f78eaa006a77f62ea50113227ef885c2)

but should probably open an issue with rxjs

Reply:
Yes, it's an issue on their side.